color.js

```diff
@@ -90,7 +90,7 @@ var color = {
 			
 			// Make sure it is always the right length
 			if(color[i].length === 1) {
-				color[i] = color[i] + color[i];
+				color[i] = "0" + color[i];
```

Comment:
Ahaha yeah I saw you wrote this 6 years ago! I'll submit another pull request later today with the min file as well.